runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/OrchestratorRequest.java

```diff
@@ -0,0 +1,52 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import io.netty.buffer.ByteBuf;
+import lombok.Getter;
+
+import org.corfudb.format.Types.OrchestratorRequestType;
+import org.corfudb.protocols.wireprotocol.ICorfuPayload;
+
+import static org.corfudb.format.Types.OrchestratorRequestType.ADD_NODE;
+
+/**
+ * A message container that encapsulates all the orchestrator's
+ * requests.
+ *
+ * Created by Maithem on 10/25/17.
+ */
+
+public class OrchestratorRequest implements ICorfuPayload<OrchestratorRequest> {
+
+    @Getter
+    final public Request request;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)

Reply:
done